tests/e2e/utils/constants.ts

```diff
@@ -10,3 +10,8 @@ export const STORAGE_STATE_PATH = path.join(
 	process.cwd(),
 	'artifacts/storage-states/admin.json'
 );
+
+// User roles file paths
+export const adminFile = '.auth/admin.json';
+export const customerFile = '.auth/customer.json';
+export const guestFile = '.auth/guest.json';
```

Comment:
We are saving these at the project root, but the `.gitignore` only ignores things in `e2e`. We should update these paths.

Reply:
@opr, when I run the tests locally, these files are created in `tests/e2e/.auth/`, not on the project root.